package.json

```diff
@@ -32,6 +32,7 @@
     "coveralls": "npm run test && cat coverage/lcov.info | coveralls"
   },
   "dependencies": {
-    "moment": "2.21.0"
+    "moment": "2.21.0",
+    "moment-timezone": "0.5.17"
```

Comment:
This is going to increase the download size for anyone using this package pretty dramatically. `moment-timezone` is not small.